### PR TITLE
Modifed dashboard as per feedback

### PR DESCRIPTION
**Pull request by vinayagg**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: jfrog_platform/assets/dashboards/jfrog_artifactory_overview.json [hunk 1]
```diff
@@ -1,40 +1,43 @@
 {
+    "author_info": {
+        "author_name": "Datadog"
+    },
     "board_title": "Jfrog Artifactory Overview",
-    "description": "This dashboard provides usage statistics overview of JFrog Artifactory logs. Included are also some high level views of audit, request and application logs. \n\nA detailed blog post can be found here: [Track JFrog Platform Performance with Datadog Analytics](https://jfrog.com/blog/track-jfrog-platform-performance-with-datadog-analytics/). \n\nSource code is also available in [Github](https://github.com/jfrog/log-analytics/tree/master/log-vendors/datadog)",
+    "description": "This dashboard provides usage statistics overview of JFrog Artifactory logs. Included are also some high level views of audit, request and application logs. \n\nA detailed blog post can be found here: [Track JFrog Platform Performance with Datadog Analytics](https://jfrog.com/blog/track-jfrog-platform-performance-with-datadog-analytics/). \n\nSource code is also available in [Github](https://github.com/jfrog/log-analytics/tree/master/log-vendors/datadog) (cloned)",
```
Comment (ChristineTChen) [inline jfrog_platform/assets/dashboards/jfrog_artifactory_overview.json:6]:
```suggestion
    "description": "This dashboard provides usage statistics overview of JFrog Artifactory logs. Included are also some high level views of audit, request and application logs. \n\nA detailed blog post can be found here: [Track JFrog Platform Performance with Datadog Analytics](https://jfrog.com/blog/track-jfrog-platform-performance-with-datadog-analytics/). \n\nSource code is also available in [Github](https://github.com/jfrog/log-analytics/tree/master/log-vendors/datadog)",
```

File: jfrog_platform/manifest.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   "integration_id": "jfrog-platform",
   "assets": {
     "dashboards": {
-      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
+      "Jfrog Artifactory Overview": "assets/dashboards/jfrog_artifactory_overview.json"
```
Comment (ChristineTChen) [inline jfrog_platform/manifest.json:27]:
```suggestion
      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_overview.json"
```
Let's only update the payload. Changing the keys here may have impact on duplicate dashboards being synced.

**Review by ChristineTChen — CHANGES_REQUESTED** [review]
Let's keep the name of the dashboard JSON file the same. The file names have no impact on the title.

**Comment by vinayagg** [comment]
Hi @ChristineTChen , I accepted your suggestions, but the checks still fail. Can you look into this further?

**Comment by ChristineTChen** [comment]
Hi @vinayagg, can you pull latest master and rebase your branch? 

There will be a conflict with the dashboard JSON file because we had to update dashboards with the `/dash` endpoint payload. With your latest dashboard, you can export it again and replace `assets/dashboards/jfrog_artifactory_dashboard.json`. For updating a dashboard, let's only update the contents and keep filename the same.

Thanks!

**Comment by vinayagg** [comment]
@ChristineTChen Modified as you requested. Still windows build passes but linux build is failing.
It is talking about some new /dashboard format. We are not sure what this means as we exported our dashboard just a week ago. Has the format changed since then?

**Comment by ChristineTChen** [comment]
Hi @vinayagg, apologies for the delayed response. I'm going to retrigger the CI and see what errors you were getting. The new `/dash` endpoint is available in the `ddev meta dash export` command if you update ddev CLI.

File: jfrog_platform/manifest.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
   "integration_id": "jfrog-platform",
   "assets": {
     "dashboards": {
-      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
+      "Jfrog Artifactory Overview": "assets/dashboards/jfrog_artifactory_dashboard.json"
```
Comment (ChristineTChen) [inline jfrog_platform/manifest.json:27]:
```suggestion
      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
```

Let's avoid renaming the key

**Review by ChristineTChen — CHANGES_REQUESTED** [review]

**Comment by vinayagg** [comment]
Hello @ChristineTChen I accepted your change suggestion. The linux pipelines are still failing. Can you have another look? Thank you.

**Comment by ChristineTChen** [comment]
Hi @vinayagg , did you update ddev and export the dashboard? It looks like the payload is using the older format.

`python -m pip install --upgrade "datadog-checks-dev[cli]"`

After upgrading ddev, you can run `ddev -e meta dash export <DASHBOARD_URL> jfrog_platform` to export the dashboard

**Comment by ChristineTChen** [comment]
@vinayagg, closing this PR in favor of https://github.com/DataDog/integrations-extras/pull/730 so we can get this dashboard updated + synced

**Pull request closed without merging**